dashboard/static/js/analytics.js

```diff
@@ -1,3 +1,17 @@
+var ga_id = document.getElementById("push-dashboard").dataset.ga_id;
+var ga_filename = document.getElementById("push-dashboard").dataset.ga_filename;
```

Comment:
Looks like `dataset` is [only supported by the latest version of IE](http://caniuse.com/#search=dataset). On the one hand, maybe we're not worried about lower versions of IE. On the other hand, we'll have a hard time judging if we should be worried about them if we can't get analytics from them. Up to you.